### ISSUE TITLE
Submit buttons in the found and lost pages forms are not well organised

### ISSUE THREAD

**Issue by shrinishant** [Volley-Freak/gotLost#23]
The submit buttons on the found and lost pages needs to be organized well to fit the the form tag/div.

**Comment by Volley-Freak** [comment]
@shrinishant sure...do it for contact and registration form also if possible.

**Comment by shrinishant** [comment]
sir, I've requested a pr for the required changes. Please check them out. 

Since, the registration page is not yet finished with styling so just can't do the same for this page. 

Can I also please contribute to registration page?

**Comment by Mihir008** [comment]
sir i can do for contact and registration please assign me this work. can i contribute ??

**Comment by Volley-Freak** [comment]
Hello @Mihir008, thank you for stepping ahead, but @shrinishant was already working on the issue let him fix this only. But you can contribute to some other issues.

**Comment by Volley-Freak** [comment]
> sir, I've requested a pr for the required changes. Please check them out.
> 
> Since, the registration page is not yet finished with styling so just can't do the same for this page.
> 
> Can I also please contribute to registration page?

Yes, you can I have added a link for the registration page and the CSS of the registration page is also completed.

**Comment by Priya1511jain** [comment]
sir can you assign this task for me.
